index.js

```diff
@@ -102,35 +102,55 @@ class ShopifyToken {
 
   /**
    * Verify the hmac returned by Shopify.
+   * The query can either be passed as a string or an object. Using a string has the advantage that order is guaranteed;
+   * if you use an object, then the order is not guaranteed so the hmac may erroneously fail to be verified.
```

Comment:
This is not true. Parameters are sorted when using an object so order is guaranteed.

Reply:
Ah, you're right. I'll make a new PR with the other changes from this one.